build.sbt

```diff
@@ -57,6 +57,17 @@ commands += Command.command("mima") { s =>
   s"very mimaReportBinaryIssues" ::
   s
 }
+commands += Command.command("ci-metac") { s =>
+  val out = file("target/scala-library")
+  if (!out.exists()) {
+    IO.unzipURL(
+      new URL(s"https://github.com/scala/scala/archive/v$LatestScala212.zip"),
+      toDirectory = out,
+      filter = s"scala-$LatestScala212/src/library/*"
+    )
+  }
+  "testsJVM/test:runMain scala.meta.tests.semanticdb.MetacScalaLibrary" :: s
```

Comment:
Can we also delete `target/scala-library` afterwards? Or maybe create this in a temporary directory?

Reply:
Dowloading+unzipping takes a while to run and if it was deleted on every run then that would slow when running this multiple times locally. We only unzip `src/library` so it's not that bloated. I prefer to keep the directory.